docs/lincheck-test-tutorial.md

```diff
@@ -32,17 +33,20 @@ repositories {
 
 dependencies {
    // Lincheck dependency
-   implementation "org.jetbrains.kotlinx:lincheck:2.12"
+   implementation "org.jetbrains.kotlinx:lincheck:2.13"
    
    // This dependency will allow you to work with kotlin.test and JUnit:
    testImplementation "org.jetbrains.kotlin:kotlin-test"
 }
 ```
 
-## Create a test
+## Write your first Lincheck test
  
 Create `CounterTest.kt` file in the directory `src/test/kotlin` and put the following code there:
 
+TODO: should we include imports in the code blocks? 
```

Comment:
Yes, I will add imports here. We will later omit imports (and any distractive code) in the runnable samples on the site using publication plugin, like this:

```
import org.jetbrains.kotlinx.lincheck.annotations.Operation
import org.jetbrains.kotlinx.lincheck.check
import org.jetbrains.kotlinx.lincheck.strategy.stress.StressOptions
import org.jetbrains.kotlinx.lincheck.verifier.VerifierState
import org.junit.Test

//sampleStart
class CounterTest {
   private val c = Counter() // initial state

   // operations on the Counter
   @Operation fun inc() = c.inc()
   @Operation fun get() = c.get()

   @Test // JUnit
   fun test() = StressOptions().check(this::class) // the magic button
}
//sampleEnd
{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
```

And the runnable sample will contain code between  `//sampleStart` `//sampleEnd` only.

Reply:
Could you please insert //sampleStart and //sampleEnd everywhere?

Reply:
//sampleStart and //sampleEnd works for hiding code lines only in runnable code snippets. This snippet cannot be runnable, it doesn't have an output

Reply:
@danil-pavlov I expected this to be a runnable code snippet as it is a JUnit test, we have a ton of similar snippets all over the guide. What is the best way to make them runnable?

Reply:
We'll look into it with the website team. Probably, we can find a solution